File: python/src/adapter/driving/well_known_test_exception_controller.py
import logging

from fastapi import APIRouter

logger = logging.getLogger(__name__)
router = APIRouter()


@router.get("/.well-known/test/exception")
def test_exception():
    """测试异常"""
    try:
        1 / 0
    except Exception as ex:
        raise Exception("测试Exception") from ex

File: dotnet/src/adapter/driving/WellKnownTestExceptionController.cs
using log4net;

using Microsoft.AspNetCore.Mvc;

namespace DddExample.Adapter.Driving
{
    /// <summary>
    /// 测试异常 Well Known Controller
    /// </summary>
    public class WellKnownTestExceptionController : ControllerBase
    {
        private static readonly ILog logger = LogManager.GetLogger(typeof(WellKnownTestExceptionController));

        /// <summary>
        /// 测试异常处理
        /// </summary>
        [HttpGet("/.well-known/test/exception")]
        public Result TestException()
        {
            throw new Exception("测试异常");
        }
    }
}
